calcom/cal.com
# fix: 15636 Sort order of team event types incorrect

## What does this PR do?



- Fixes #15636 (GitHub issue number)
- Fixes CAL-XXXX (Linear issue number - should be visible at the bottom of the GitHub issue description)



## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?


- on event types page create new event types
- Sort event types on eventypes page
- go to team page
- even types should match the order on eventypes page.
## Checklist
